src/esw/brushed_motors/Controller.cpp

```diff
@@ -76,12 +101,84 @@ void Controller::moveOpenLoop(float input) {
 
         I2C::transact(deviceAddress, motorIDRegMask | OPEN_PLUS_OP, OPEN_PLUS_WB,
                       OPEN_PLUS_RB, buffer, UINT8_POINTER_T(&angle));
-        currentAngle = (float) ((((float) angle / quadCPR) * 2 * M_PI) - M_PI);
+        currentAngle = (float) (((float) angle / quadCPR) * 2 * M_PI);
     } catch (IOFailure& e) {
         ROS_ERROR("moveOpenLoop failed on %s", name.c_str());
     }
 }
 
+// REQUIRES: nothing
+// MODIFIES: isCalibrated
+// EFFECTS: asks the MCU if it is calibrated
+void Controller::askIsCalibrated() {
+    try {
+        makeLive();
+
+        uint8_t calibration_status;
+        I2C::transact(deviceAddress, motorIDRegMask | IS_CALIBRATED_OP, IS_CALIBRATED_WB,
+                      IS_CALIBRATED_RB, nullptr, UINT8_POINTER_T(&calibration_status));
+        
+        isCalibrated = calibration_status;
+
+    } catch (IOFailure& e) {
+        ROS_ERROR("askIsCalibrated failed on %s", name.c_str());
+    }
+}
+
+// REQUIRES: nothing
+// MODIFIES: nothing
+// EFFECTS: enables or disables limit switches
+void Controller::enableLimitSwitches(bool enable) {
+    try {
+        makeLive();
+
+        bool enableLimitA = limitAEnabled && enable;
```

Comment:
so limitAEnabled and limitBEnabled are like hardware constants. They are setup in the config and are used to show that a motor is able to support a limit switch.
If you AND (&&) it with "enable", this shows what the motor will use.

IDK if this explanation was clear.

Reply:
so if the user doesn't want the limit switches (enable = 0), then the motor will not use the limit switches.
If the user wants the limit switches (enable = 1), then the motor will only use the limit switches that we say it owns in the config (limitAEnabled and limitBEnabled)

Reply:
Yeah that makes sense, I was incorrect. I stand by saying there's an issue with the naming convention though, this is a little confusing since the word "enable" is overloaded. Could we say limitAPresent or something else to denote when there is a physical limit switch there that can be enabled or disabled?

Reply:
Renaming it to limitAPresent